tests/foreman/cli/test_contenthost.py

```diff
@@ -567,115 +559,38 @@ def setUp(self):
 
```

Comment:
If that's the idea, then we have benefit in saving time by using the same org/env/CV, but the drawbacks are increased code complexity and changing the code style. I've quickly searched through code and couldn't found classes, which are using both ```setUpClass()``` and ```setUp()```, so it's definitely not the commonly-used approach. @elyezer , could you please share your thoughts on that?